src/ui/progress.js

```diff
@@ -32,12 +32,12 @@ class Circle {
         .progress(progress => {
           // if we are still listening to this one
           if (file === this.observed) {
-            return this.renderer.setValue(selectorFn(progress))
+            this.renderer.setValue(selectorFn(progress))
```

Comment:
Returns aren't needed actually, they're here because of coffeescript -> js compilation